cogs/admin.py

```diff
@@ -39,40 +41,44 @@ async def sql(self, ctx, *, query):
         text = "|"
 
         for i in keys:
-            text += " {} |".format(str(i).ljust(key_length[i]))
+            text += f" {str(i).ljust(key_length[i])} |"
 
         text += "\n" + '-' * len(text)
 
         for row in result:
             newtext = "\n|"
             for key in keys:
-                newtext += " {} |".format(str(row[key]).ljust(key_length[key]))
+                newtext += f" {str(row[key]).ljust(key_length[key])} |"
 
             # -6: Account for code block
             if len(text) + len(newtext) >= 2000 - 6:
-                await ctx.send("```{}```".format(text))
+                await ctx.send(f"```{text}```")
                 text = ""
 
             text += newtext
 
-        await ctx.send("```{}```".format(text))
+        await ctx.send(f"```{text}```")
 
     @commands.command(aliases=['purge'])
     @commands.cooldown(2, 600, type=commands.BucketType.default)
     @commands.has_permissions(administrator=True)
     async def clear(self, ctx, amount=10):
+        """Removes up to 20 messages"""
```

Comment:
"Removes a given amount of messages"

Reply:
"Removes a given amount (20 max) of messages"?